app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ActiveRecord::Base
+  has_many :bookings
+  has_secure_password
+
+  validates_presence_of :name
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.